tr55/model.py

```diff
@@ -105,6 +105,16 @@ def simulate_cell_day(precip, evaptrans, cell, cell_count):
     The return value is a dictionary of runoff, evapotranspiration, and
     infiltration as volumes of water.
     """
+    def clamp(runoff, et, inf, precip):
```

Comment:
Can you add a code comment that explains that runoff, et and inf are scaled proportionally down in situations where the original sum would exceed precipitation?  I'd like that fact to jump out at someone who was skimming the code, especially since the `et` value which is normally fixed would be shifted in this case.